iModelJsNodeAddon/JsInteropDgnDb.cpp

```diff
@@ -509,11 +509,20 @@ DgnDbStatus JsInterop::SimplifyElementGeometry(DgnDbR db, Napi::Object simplifyA
 /*---------------------------------------------------------------------------------**//**
 * @bsimethod
 +---------------+---------------+---------------+---------------+---------------+------*/
-void JsInterop::UpdateProjectExtents(DgnDbR dgndb, BeJsConst newExtents) {
+void JsInterop::UpdateProjectExtents(DgnDbR dgndb, BeJsConst newExtents, bool fromChangesetAppliedEvent) {
     auto& geolocation = dgndb.GeoLocation();
     AxisAlignedBox3d extents;
     extents.FromJson(newExtents);
-    geolocation.SetProjectExtents(extents);
+    if (fromChangesetAppliedEvent) {
+        // Check if our extents have actually changed. If not, return early.
+        if (geolocation.GetProjectExtents().IsEqual(extents))

```

Comment:
Couldn't we just always check if really changed and not bother with "fromChangesetAppliedEvent"?

Reply:
Any opinions on what that tolerance should be? I pushed '.01' up just now.

Youre right, I probably dont need to check fromChangesetAppliedEvent before checking if itsEqual. I just figured this function has been working fine before checking for isEqual so might as well keep it to the fromChangesetAppliedEvent.

Changed it to check isEqual always

Reply:
I'd compare with something like DoubleOps::SmallMetricDistance(). 10 mm is significate and can't be attributed to fuzz...